docs/http-api/zone.rst

```diff
@@ -37,24 +38,19 @@ Comments are per-RRset.
 
   Switching ``dnssec`` to ``true`` (from ``false``) sets up DNSSEC signing
   based on the other flags, this includes running the equivalent of
-  ``secure-zone`` and ``rectify-zone``. This also applies to newly created
-  zones. If ``presigned`` is ``true``, no DNSSEC changes will be made to
-  the zone or cryptokeys.
-
-  ``dnssec``, ``nsec3narrow``, ``nsec3param``, ``presigned`` are not yet implemented.
+  ``secure-zone`` and ``rectify-zone`` (if ``api_rectify`` is set to "1").
+  This also applies to newly created zones. If ``presigned`` is ``true``,
```

Comment:
the mismatch of 1 and true is quite ugly here. I think API consumers should not carry the burden of dealing with different truthiness values

Reply:
I think I do make a bool out of, but it is wrongly documented

Reply:
the code below indeed confused me. please check whats actually there and i think boolifying it is a good idea (:

Reply:
The POST/PUT and GET for `api_rectify` were not consistent. It is fully boolified on the API level in the upcoming push.